README.md

```diff
@@ -10,12 +10,18 @@ Minimal doc product is a small repo that is designed to give you an starting poi
 
 ### To simply run locally
 
-1. Make sure you have permissions for https://rubygems.pkg.github.com/nexmo. Instructions are [here](https://docs.github.com/en/packages/using-github-packages-with-your-projects-ecosystem/configuring-rubygems-for-use-with-github-packages)
-2. Clone this repo.
-3. Change into cloned directory.
-4. Check the `Gemfile` for the correct Ruby version.
-5. Select the correct version of Ruby, for example:
+`NOTE: Commands are to be executed on Linux, Mac, and Windows(using Powershell)`
 
+1. Make sure you have permissions for https://rubygems.pkg.github.com/nexmo. Instructions are [here](https://docs.github.com/en/packages/using-github-packages-with-your-projects-ecosystem/configuring-rubygems-for-use-with-github-packages)
+2. Clone the repo by running the command
```

Comment:
If breaking like this use colon at end of sentence and then have a blank line before the code block.

Reply:
Sure, updated.